src/rulesets/obsoleteAttr.js

```diff
@@ -60,4 +60,10 @@ module.exports = [
         error : 'align attribute in <h1>...<h6> is completely obsolete, and must not be used by authors.',
         solution : 'Use CSS instead.'
     },
+    {
+        tags : 'time',
```

Comment:
Can you please show the link refers to this rule

Reply:
https://www.tutorialspoint.com/html5/html5_deprecated_tags.htm

Please refer the Deprecated Attributes column. I use this site for my academic purposes :)

Reply:
@shalithasuranga  Please refer this article on https://www.sitepoint.com/5-obsolete-features-html5/ 
Thanks

Reply:
@ChathuminaVimukthi  it is very old article. I cannot find `pubdate` attribute from here https://www.w3.org/TR/html5/obsolete.html (The html5 reference for obsolete attributes)

Reply:
You are free to pickup new obsolete attribute from https://www.w3.org/TR/html5/obsolete.html and update your branch I guess @ChathuminaVimukthi Better to use the w3c official sources . Thanks

Reply:
@shalithasuranga I'm sorry for the mess. I didn't refer the w3. May I change the commits?

Reply:
No worries @ChathuminaVimukthi  You can add new valid rule from w3c and update this

Reply:
@shalithasuranga I updated the rules :) Thanks for the advices